Project.toml

```diff
@@ -32,7 +33,6 @@ SimplePosets = "b2aef97b-4721-5af9-b440-0bad754dc5ba"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsFuns = "4c63d2b9-4356-54db-8cca-17b64c39e42c"
-SymPy = "24249f21-da20-56a4-8eb1-6a02cf4ae2e6"
```

Comment:
It's probably a good idea to keep SymPy under the [extras] section for testing purposes. Here's how the project file  looks for MCM, all the packages listed under extras and targets/test can be used while testing

```
[extras]
ControlSystems = "a6e380b2-a6ca-5380-bf3e-84a91bcd477e"
ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
Measurements = "eff96d63-e80a-5855-80a2-b1b0885c5ab7"
Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
SLEEFPirates = "476501e8-09a2-5ece-8869-fb82de89a1fa"
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"

[targets]
test = ["ControlSystems", "ForwardDiff", "Measurements", "SLEEFPirates", "Test", "Plots"]
```
They will only be installed if tests are run, not when installing the package

Reply:
Great point, thanks. As I understand, this has changed recently, and the "modern" approach is to have separate `Project.toml`s for the project itself, `docs`, and `test`. I'll add SymPy to the... I want to say "latter two", is that a thing? :)

Reply:
Using a separate project file for docs is common. I haven't seen it for tests before but that might work as well, although I can't find any info about it at https://julialang.github.io/Pkg.jl/v1/toml-files/
In either case, not automatically installing sympy when Soss is installed would be great

Reply:
Yeah, I haven't seen it for tests either. I don't think it would be necessary due to `extras`.